lavuelib/sourceWidget.py

```diff
@@ -1752,6 +1750,7 @@ def configure(self, configuration):
         :param configuration: configuration string
         :type configuration: :obj:`str`
         """
+        print(configuration)
```

Comment:
Interestingly this part of code is never called in my test. And the format of the configuration indicated in this part of code is different from what it is in the other parts. I assume this part is obsolete and perhaps needs to be removed in the future.

Reply:
it is called from "LiveViewer.__applyoptions()" in two cases:
-  when you start lavue with -c option e.g.
       lavue -s nxsfile -c /tmp/myfile.nxs://entry/data/pilatus,1
- when you use Tango Control System (Epics is not supported in this case)  you can set a LavueController  server and change lavue configuration with its LavueController.LavueState attrubute.

A number in the configuration above corresponds to a staking dimension of the dataset. We can add a second numberfor the frame number you added.